bali/core.py

```diff
@@ -88,14 +89,18 @@ def initialize(settings):
         logging.config.dictConfig(settings.LOGGING_CONFIG)
 
     if not hasattr(settings, 'BALI_QUEUE'):
-        setattr(settings, 'BALI_QUEUE', 'event_default_queue')
+        settings.Config.__setattr__(
+            settings, 'BALI_QUEUE', 'event_default_queue'
```

Comment:
Maybe the default queue's name should be `{service_abbr}.events`

eg:
    user.events
    third_data.events